adoc/bp_postgresql_database_migration.adoc

```diff
@@ -39,44 +43,43 @@ smdba-1.5.8-0.2.3.1.x86_64
 
 [IMPORTANT]
 ====
-Postgresql 9.6 requires smdba version 1.5.8 or higher
+PostgreSQL 9.6 requires smdba version 1.5.8 or higher
 ====
 
 [WARNING]
 ====
 Always create a database backup before performing a migration
 ====
 
-
-$$.$$
 The database migration begins by executing the following command:
 
 ----
 $> /usr/lib/susemanager/bin/pg-migrate.sh
 ----
 
 
-The [path]``pg-migrate.sh``
- script will automatically perform the following operations:
+The [path]``pg-migrate.sh`` script will automatically perform the following operations:
 
 * Stop spacewalk services
 * Shut down the running database
-* Check if postgresql 9.6 is installed and install it if not already present
-* Switch from postgresql 9.4 to postgresql 9.6 as the new default
+* Check if PostgreSQL 9.6 is installed and install it if not already present
+* Switch from PostgreSQL 9.4 to PostgreSQL 9.6 as the new default
 * Initiates the database migration
-* Creates a postgresql configuration file tuned for use by SUSE Manager (The reason for the latest version of smdba)
+* Creates a PostgreSQL configuration file tuned for use by SUSE Manager (The reason for the latest version of smdba)
 * Start both the database and spacewalk services
 
 
 [NOTE]
 ====
-Please note that during the migration the data directory of the database is copied for use by the postgresql 9.6.
+During the migration the data directory of the database is copied for use by the PostgreSQL 9.6.
 This results in temporarily doubling the amount of required disk space.
 In case of a failure, the migration script will attempt a restore to the original state.
 After a successful migration, you may safely delete the old database directory (renamed to /var/lib/pgsql/data-pg94) to reclaim lost disk space.
 ====
 
-== Performing a Fast Migration
+
+
+==== Performing a Fast Migration
```

Comment:
We're deep in the heading levels now. Should this be a procedure heading?

Reply:
redone